web/static/css/app.css

```diff
@@ -165,14 +171,34 @@ a {
 }
 
 .bottom__offset {
-  background-color: #47B9B4;
+  background-color: white;
   padding: 20px;
 }
 
 .bottom__title {
   font-size: 35px;
 }
 
+.bottom__back-link {
+  font-size: 16px;
+  font-weight: 700;
+  color: #9b9b9b;
+  padding: 10px;
+}
+
+.bottom__back-link a {
+  color: #9b9b9b;
```

Comment:
And the fact that you use the same colour in many places is an extra reason why you should use a preprocessor like Sass.